environment/deployments/roundtable/main.tf

```diff
@@ -21,6 +21,32 @@ module "iam_admin" {
   member                  = "gcp-${var.application_name}-administrators@lsst.cloud"
 }
 
+resource "google_service_account" "git_lfs_rw_sa" {
+  account_id   = "git-lfs-rw"
+  display_name = "Git LFS (RW)"
+  description  = "Terraform-managed service account for Git LFS RW access"
+  project      = module.project_factory.project_id
+}
+
+resource "google_service_account_iam_member" "git_lfs_rw_sa_wi" {
+  service_account_id = google_service_account.git_lfs_rw_sa.name
+  role               = "roles/iam.workloadIdentityUser"
+  member             = "serviceAccount:${module.project_factory.project_id}.svc.id.goog[git-lfs/rw]"
+}
+
+resource "google_service_account" "git_lfs_ro_sa" {
+  account_id   = "git-lfs-ro"
+  display_name = "Git LFS (RO)"
+  description  = "Terraform-managed service account for Git LFS RO access"
+  project      = module.project_factory.project_id
+}
+
+resource "google_service_account_iam_member" "git_lfs_ro_sa_wi" {
+  service_account_id = google_service_account.git_lfs_ro_sa.name
+  role               = "roles/iam.workloadIdentityUser"
+  member             = "serviceAccount:${module.project_factory.project_id}.svc.id.goog[git-lfs/ro]"
+}
+
```

Comment:
Huh.  With only a single service account (assuming we solve the keys problem) how do we enforce separate read-only and read-write access?  We want the Giftless instance running at `-rw` to be able to write to the bucket, but the other one only to be able to read from it.

Reply:
(the dumb way is, well, make it two deployments, each with its own account)

Reply:
Oh, I see, there are two deployments, and the part before the slash is the namespace so the service accounts are named `ro` and `rw`. I was assuming you would just use one Google service account because I'd forgotten there were two deployments. I suppose we could use separate service accounts for each deployment; I'm not sure it buys that much over using the same one, but sure, may as well.

I think my remaining request would be to call the service accounts something more informative than `ro` and `rw`. Maybe `giftless-ro` and `giftless-rw`.

Reply:
About the slash: the only other model I found was in science-platform/main.tf:

```
resource "google_service_account_iam_member" "hips_sa_wi" {
  service_account_id = google_service_account.hips_sa.name
  role               = "roles/iam.workloadIdentityUser"
  member             = "serviceAccount:${module.project_factory.project_id}.svc\
.id.goog[hips/hips]"
}
```

But I don't really know what that is about.  Should I just drop the bracketed part?

Reply:
Ah, OK, so since the namespace is "giftless", "[giftless/git-lfs-rw]" (resp. "ro") is what I want.

Reply:
The bracketed part is now "[giftless/git-lfs-ro]" or "rw", which I believe to be correct.  The Terraform Plan output looks correct to me.